### DISCUSSION TITLE
Errore richiamo Facades

### DISCUSSION THREAD

**Discussion by Askancy** [murdercode/laravel-shortcode-plus#56] [General]
Nella documentazione riportate:

`use Murdercode\ShortcodePlus\Facades\ShortcodePlus;`

che però non esiste, non dovrebbe essere così?

`use Murdercode\LaravelShortcodePlus\Facades\LaravelShortcodePlus;`

**Comment by murdercode** [comment]
Giusto, fixata grazie :)
